src/harmonic/phonon_h0.cpp

```diff
@@ -561,6 +696,15 @@ void PhononH0::shortRangeTerm(Eigen::Tensor<std::complex<double>, 4> &dyn,
       }
     }
   }
+    //for (int nb = 0; nb < numAtoms; nb++) {
```

Comment:
@anjohan there's a bunch of commented print statements in this function, do we want to keep them still for some reason?

Reply:
We discussed and decided to keep them.